lib/spack/spack/modules.py

```diff
@@ -428,13 +428,23 @@ def header(self):
     def module_specific_content(self, configuration):
         return tuple()
 
+    def verbose_autoload(self):
```

Comment:
Sorry... testing with tcl make me think the original place would be a classmethod of EnvModule, in order to get config properly:

+    def verbose_autoload(self):
+        #lthis work just for lmod#configuration = _module_config.get('lmod', {})
+        configuration = _module_config.get(self.name, {})
+        return configuration.get('verbose_autoload', True)